bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusEnergyMeter.xml

```diff
@@ -15,6 +15,8 @@
 
 		<channels>
 			<channel id="power" typeId="power"/>
+			<channel id="energyToday" typeId="energyToday"/>
+			<channel id="energyThisMonth" typeId="energyThisMonth"/>
```

Comment:
As you add new channels, please provide the upgrade instructions for users using thing created through UI.
Let me know if "providing upgrade instructions" is not clear for you.

Reply:
As a metter of fact, "providing upgrading instructions" looks grim to me, Please give me any tip

Reply:
@jlaur : was it documented somewhere or should we still provide a link to an example ?

Reply:
See https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Reply:
That's cool  I was not aware that it was done.

Reply:
ok done